packages/cactus-plugin-satp-hermes/src/main/typescript/plugin-satp-hermes-gateway.ts

```diff
@@ -279,7 +279,11 @@ export class SATPGateway implements IPluginWebService, ICactusPlugin {
             Crash: "v02",
           },
         ],
-        supportedDLTs: [SupportedChain.FABRIC, SupportedChain.BESU],
+        supportedDLTs: [
+          SupportedChain.FABRIC,
+          SupportedChain.BESU,
+          SupportedChain.ETHEREUM,
```

Comment:
Please change to EVM (the connector allows to connect to any evm chain)

Reply:
Done ! Thank you